src/check.ts

```diff
@@ -7,6 +7,26 @@ import {plural} from './render';
 
 const USER_AGENT = `${pkg.name}/${pkg.version} (${pkg.bugs.url})`;
 
+interface AnnotationContents {
```

Comment:
This one is actually called `Message` at [cargo sources](https://github.com/rust-lang/cargo/blob/d764fff2da618b24fbf506229509e3d183050c2e/src/cargo/util/machine_message.rs#L9), maybe it is worth to rename it into smth like `CargoMessage`, as it is not an annotation yet?